### PR TITLE
Update/nar2021.004

### PR DESCRIPTION
**Pull request by esguerra**
Sorry for the non-squashed request.
Had forgotten to turn-on cache again.

### CODE REVIEW

File: home/views.py [hunk 1]
```diff
@@ -19,9 +22,10 @@ def index(request):
     context['google_analytics_key'] = settings.GOOGLE_ANALYTICS_KEY
 
     if settings.GOOGLE_ANALYTICS_API:
-         # Based on https://developers.google.com/analytics/devguides/reporting/core/v3/quickstart/service-py
-        from apiclient.discovery import build
-        from oauth2client.service_account import ServiceAccountCredentials
+        # Based on https://developers.google.com/analytics/devguides/reporting/core/v3/quickstart/service-py
+        # What in the HELL! were these two imports doing here?
```
Comment (AJK-dev) [inline home/views.py:26]: Comment can (imho) be removed together with the imports. 
My best guess is that the import was put there to be prevent the import when the GAA setting is not present.

File: protwis/settings.py [hunk 2]
```diff
@@ -191,3 +191,8 @@
         }
     }
 }
+
+# Note that https://www.django-rest-framework.org/community/3.10-announcement
```
Comment (AJK-dev) [inline protwis/settings.py:195]: Good to know!

File: structure/views.py [hunk 3]
```diff
@@ -489,7 +489,8 @@ def get_context_data (self, **kwargs):
 		context['class_f_options']['label_free'] = [1,]
 		#json.dump(class_f_data.get_nodes_dict('crystalized'), open('tree_test.json', 'w'), indent=4)
 		context['class_f'] = json.dumps(class_f_data.get_nodes_dict('crystals'))
-		class_t2_data = tree.get_tree_data(ProteinFamily.objects.get(name='Class T (Taste 2)'))
+                class_t2_data = tree.get_tree_data(ProteinFamily.objects.get(name='Class T (Taste 2)'))
```
Comment (AJK-dev) [inline structure/views.py:492]: Doesn't the newly introduced indentation break the execution?
The commented line below also doesn't have to be added imho.

**Review by AJK-dev — CHANGES_REQUESTED** [review]
Few minor comments.

Please double check the new indentation in structure/views.py. I think the execution will break due to that. Otherwise everything is looking sharp.

File: home/views.py (hunk 1 above)
Comment (esguerra) [reply home/views.py:26]: Yes. You're right.
Just noticed.
Agree with the comments.

File: structure/views.py (hunk 3 above)
Comment (esguerra) [reply structure/views.py:492]: You're right.
Fixing.

**Review by AJK-dev — APPROVED** [review]

**Pull request merged**